### PR TITLE
Do not use ES6 Default Export Where Upstream Uses CommonJS Import

### PR DESCRIPTION
**Pull request by NickGerleman**
This was dervied from a quick search for anything in vnext with default export
which ends with ".windows" or ".win32" . The files with exports changed are
imported using CommonJS in upstream Facebook code. These are all currently
benign since we don't run FB's code here, but we should keep them in sync.

Validated TextExample module structure looks correct when imported, though it
still blows up due to unrelated issues (see #3876).

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/react-native-windows/pull/3899)

### CODE REVIEW

File: vnext/src/RNTester/js/RNTesterList.windows.ts [hunk 1]
```diff
@@ -238,7 +238,7 @@ const APIExamples: Array<IRNTesterExample> = [
   },*/
 ];
 
-const Modules: {[key: string]: IRNTesterModule} = {};
+const Modules: { [key: string]: IRNTesterModule } = {};
```
Comment (kmelmon) [inline vnext/src/RNTester/js/RNTesterList.windows.ts]: >{ [key: string]: IRNTesterModule } [](start = 15, length = 34)

The CI loop is unhappy about this change.  Are you using your own prettier formatter tool?

I've been getting in the habit of running yarn lint:fix from the vnext directory before pushing changes.

Comment (NickGerleman) [reply vnext/src/RNTester/js/RNTesterList.windows.ts]: I did that for the last commit here 😶 Curious why this wasn't picked up.

VSCode is using the wrong prettier config on save on my machine right now. I should probably spend some time trying to fix that.

**Comment by NickGerleman** [comment]
@acoates-ms this PR seems to be hitting the Git issue determininistically. Did you have ideas on what's causing it, or would you be able to bypass CI here?

**Comment by acoates-ms** [comment]
https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/pipeline-options-for-git?view=azure-devops

>  When the build is queued, the branch to build is resolved to a commit ID. Then, the agent fetches the branch and checks out the desired commit. There is a small window between when a branch is resolved to a commit ID and when the agent performs the checkout. If the branch updates rapidly and you set a very small value for shallow fetch, the commit may not exist when the agent attempts to check it out. If that happens, increase the shallow fetch depth setting.

Sounds like we should just bump the depth up to 10 or something in the yml file:
https://github.com/microsoft/react-native-windows/blob/28d559f206a5e189eb22b2482629ebb4fac685e1/.ado/windows-vs-pr.yml#L604

I believe this is the number of commits that the build can handle occurring between when the pipeline was triggered and when the final build kicks off (which for the nuget one, is after all the slices have completed)

**Pull request merged**